Assets/MirageXR/Tutorial/HelpSteps/HelpSelectionModel.cs

```diff
@@ -0,0 +1,51 @@
+using Newtonsoft.Json;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using UnityEngine;
+
+namespace MirageXR
+{
+    /// <summary>
+    /// Model for all items in the help selection dialogue.
+    /// Should be serialisable.
+    /// </summary>
+    public class HelpSelectionModel
+    {
+        /// <summary>
+        /// Text that should be shown in the help popup, when the user is selecting help.
+        /// </summary>
+        [JsonProperty("selection_text")] public string SelectionText { get; set; }
```

Comment:
```suggestion
        [JsonProperty] public string SelectionText { get; set; }
```

for others too

Reply:
Why remove these? How will it parse then?

Reply:
I've just tried removing the names and it doesn't work without them.